Source/WebCore/platform/graphics/cv/GraphicsContextGLCVCocoa.mm

```diff
@@ -182,17 +182,17 @@ static TransferFunctionCV transferFunctionFromString(CFStringRef string)
 {
     if (!string)
         return TransferFunctionCV::Unknown;
-    if (CFEqual(string, kCVImageBufferYCbCrMatrix_ITU_R_709_2))
+    if (RetainPtr matrix = kCVImageBufferYCbCrMatrix_ITU_R_709_2; CFEqual(string, matrix.get()))
```

Comment:
We should just fix the static analyzer to treat CFEqual as a trivial function.

Reply:
I rebased, once the CFEqual fix is available on the smartcpp bot, this should be basically a one liner fix for GraphicsContextGLCVCocoa.